_posts/2021-03-23-world-moveit-day-2021-lighting-talks.md

```diff
@@ -14,28 +14,28 @@ categories:
 - MoveIt
 ---
 
-The 6th annual 2021 World MoveIt Day saw 90 MoveIt contributors from around the world come together to work on the open-source MoveIt project. With over 5000 lines of code written, 115 pull requests reviewed,, and 4 virtual meetups taking place on 3 continents.
+The 6th annual 2021 World MoveIt Day saw 90 MoveIt contributors from around the world come together to work on the open-source MoveIt project, with over 5000 lines of code written, 115 pull requests reviewed, and 4 virtual meetups taking place on 3 continents.
 
-Part of the event was a “lightning talk” submission. Where contributors submit a 120 second or less talk on a topic of their choice. Ideas range from what to work on to what the contributor has solved. Here are the 2021 submissions as hosted on Youtube.
+Part of the event was a “lightning talk” submission, where contributors submit a video of 120 seconds or less on a topic of their choice. Topics range from ideas about what to work on, to things that the contributor has solved. Here are the 2021 submissions as hosted on Youtube:
 
 <iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/0uXkHMpu_L8" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 
- **Boston Cleek’s talk:** ROS2 ament_cmake tutorial on how to implement a symlink install.
+ **Boston Cleek**'s talk: ROS2 ament_cmake tutorial on how to implement a symlink install
 
 <iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/IJzq8b3xLNo" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 
-**Tyler Weaver’s** talk on ROS2 Depend on Boost Component Libraries
+**Tyler Weaver**'s talk: ROS2 Depend on Boost Component Libraries
 
 <iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/Q-RIQi5ru3o" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
Here strikes the `www.youtube-nocookie.com` again.  Not a criticism of this PR, just pointing out how _double-speek_ google is about stuff like this (this does in-fact try to access a cookie).  I really hope someday the person who named this domain has to account for the lie that it propagates.